modules/etcd-operator.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * operators/operator-reference.adoc
+
+[id="etcd-cluster-operator_{context}"]
+= etcd cluster Operator
+
+[discrete]
+== Purpose
+
+The etcd cluster Operator handles the scaling of etcd and provisioning etcd dependencies such as TLS certificates. The etcd cluster Operator simplifies the disaster recovery procedure to restore to a previous cluster state, automates the addition of etcd members, provides more accurate etcd member health reporting, and reports events to assist with debugging the etcd cluster.
```

Comment:
Throughout: Odd capitalization of Operator and inconsistent naming — let's choose a consistent way to refer to the component, e.g. "the etcd operator".

Reply:
@ironcladlou right now docs follows guidance to capitalize all instances of "Operator". I'm not personally a fan, but it's what we follow. But it's consistent with how we refer to it elsewhere in the docs, for example here: https://docs.openshift.com/container-platform/4.5/backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state

Let me know if you see anywhere in the docs that it's called something other than "etcd cluster Operator".

Reply:
@ironcladlou But other than that ^, does the rest of the info in this look correct?

Reply:
>The etcd cluster Operator simplifies the disaster recovery procedure to restore to a previous cluster state, automates the addition of etcd members, provides more accurate etcd member health reporting, and reports events to assist with debugging the etcd cluster.

I would suggest a few simplifications for consideration:

>The etcd cluster Operator automates etcd cluster scaling, enables etcd monitoring and metrics, and simplifies disaster recovery procedures.

Reply:
Done, thanks!